lib/data/mailboxMetadataApi.ts

```
@@ -60,7 +60,7 @@ export const deleteMailboxMetadataApiByMailboxSlug = async (mailboxSlug: string)
 };
 
 export const testMailboxMetadataApiURL = async (mailboxSlug: string) => {
```

ESLint

[error] 'mailboxSlug' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)